src/services/signer_service.ts

```diff
@@ -158,9 +176,18 @@ export class SignerService {
             signature,
             protocolFee,
         );
-
-        const signedEthereumTransaction = await this._privateWalletSubprovider.signTransactionAsync(ethereumTxnParams);
-
+        (PrivateKeyWalletSubprovider as any)._validateTxParams(ethereumTxnParams);
```

Comment:
Add `// HACK(oskar): Using private method to validate txParams since not yet exposed by package`